side-products/merkl.md

```diff
@@ -0,0 +1,60 @@
+# Merkl, by Angle
```

Comment:
Let's have a description (like in all the other pages of the docs and try to have an emoji)